core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -179,17 +181,16 @@ class BrokerServer(
       val controllerNodes = RaftConfig.quorumVoterStringsToNodes(controllerQuorumVotersFuture.get()).asScala
       val controllerNodeProvider = RaftControllerNodeProvider(metaLogManager, config, controllerNodes)
 
-      val forwardingChannelManager = BrokerToControllerChannelManager(
+      clientToControllerChannelManager = BrokerToControllerChannelManager(
         controllerNodeProvider,
         time,
         metrics,
         config,
-        channelName = "forwarding",
+        channelName = "clientToControllerChannel",
```

Comment:
How about "controllerForwardingChannel"? I think it fits for both cases we're handling.

Reply:
Sg!